### PR TITLE
Added isObserver

### PR DESCRIPTION
**Pull request by StabbedToDeath**

### CODE REVIEW

File: src/main/java/net/rrworld/valorant/client/model/Player.java [hunk 1]
```diff
@@ -95,6 +97,14 @@ public Integer getCompetitiveTier() {
     public void setCompetitiveTier(Integer competitiveTier) {
         this.competitiveTier = competitiveTier;
     }
+	
+    public Boolean getObserver() {
```
Comment (reuhreuh) [inline src/main/java/net/rrworld/valorant/client/model/Player.java]: rename to getIsObserver()

File: src/main/java/net/rrworld/valorant/client/model/Player.java [hunk 2]
```diff
@@ -95,6 +97,14 @@ public Integer getCompetitiveTier() {
...
+        return isObserver;
+    }
+
+    public void setObserver(Boolean observer) {
```
Comment (reuhreuh) [inline src/main/java/net/rrworld/valorant/client/model/Player.java]: rename to setIsObserver()

**Review by reuhreuh — COMMENTED** [review]
Hi thanks for the PR. I try to keep getter & setters matching attribute name (getObserver() looks like its returning an object of type Observer)

**Comment by StabbedToDeath** [comment]
That's fair. I created the PR using Gits text editor instead of any IDE so it completely slipped my mind to name the method appropriately.

**Pull request merged**